### DISCUSSION TITLE
Under active development

### DISCUSSION THREAD

**Discussion by SNMS95** [mfschubert/sparsejac#2] [Q&A] [answered]
I understand that jax is experimenting with sparity. 
Does this mean that this package will be updated in the future, as and when changes are brought in JAX?

**Comment by mfschubert** [comment, marked as answer]
Hi @SNMS95, I am planning to finally get around to revisiting this code and releasing it as a package. I would hope to have this done within a few months. At that point I would also make any changes needed to retain the functionality with any newer version of Jax.
